### PR TITLE
[RELEASE ] 4.2.0-beta.1

### PR DESCRIPTION
**Pull request by Aitorbp**
## Related Issues
App:

- [ ] Added changelog files for the fixed issues in folder changelog/unreleased. More info [here](https://github.com/owncloud/android/tree/master/changelog#create-changelog-items)
_____

### CODE REVIEW

File: owncloudApp/build.gradle [hunk 1]
```diff
@@ -96,8 +96,8 @@ android {
 
         testInstrumentationRunner "com.owncloud.android.utils.OCTestAndroidJUnitRunner"
 
-        versionCode = 41000100
-        versionName = "4.1.1"
+        versionCode = 42000101
```
Comment (jesmrec) [inline owncloudApp/build.gradle:99]: This is not correct. The correct should be `41000101`.

File: owncloudApp/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -726,10 +726,18 @@
     <string name="release_notes_proceed">Proceed</string>
     <string name="release_notes_icon">Release note icon</string>
 
-    <string name="release_notes_4_1_1_title_1">Thumbnails correctly shown for every user</string>
-    <string name="release_notes_4_1_1_subtitle_1">Users with an @ in their username will now be able to see thumbnails correctly</string>
-    <string name="release_notes_4_1_1_title_2">Minor bugfixes</string>
-    <string name="release_notes_4_1_1_subtitle_2">Some bugs were fixed to improve experience in the app</string>
+    <string name="release_notes_4_2_0_title_1">"Apply to all" when many name conflicts arise</string>
+    <string name="release_notes_4_2_0_subtitle_1">A new dialog has been created where a checkbox has been added to be able to select all the folders or files that have conflicts</string>
```
Comment (jesmrec) [inline owncloudApp/src/main/res/values/strings.xml:730]: More simple: `New dialog to fix all conflicts at once`

File: owncloudApp/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -726,10 +726,18 @@
...
+    <string name="release_notes_4_2_0_title_2">"Share to" in oCIS accounts allows upload to any space</string>
```
Comment (jesmrec) [inline owncloudApp/src/main/res/values/strings.xml:731]: allows to upload

File: owncloudApp/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -726,10 +726,18 @@
...
+    <string name="release_notes_4_2_0_subtitle_2">With this improvement, shared stuff from other apps can be uploaded to any space and not only the personal one in oCIS accounts</string>
```
Comment (jesmrec) [inline owncloudApp/src/main/res/values/strings.xml:732]: Just explain the improvement, this statement `With this improvement` not take place here. Also, i'd change `stuff` for `files` . Stuff is too generic in life :D

File: owncloudApp/src/main/res/values/strings.xml [hunk 5]
```diff
@@ -726,10 +726,18 @@
...
+    <string name="release_notes_4_2_0_title_3">Deep link open app correctly</string>
+    <string name="release_notes_4_2_0_subtitle_3">Opening the app with the deep link correctly and managing if user logged or not.</string>
```
Comment (jesmrec) [inline owncloudApp/src/main/res/values/strings.xml:734]: `A link that points to the app will open the app and then, resolve the file to preview it directly`

**Review by jesmrec — CHANGES_REQUESTED** [review]

File: owncloudApp/src/main/res/values/strings.xml [hunk 6]
```diff
@@ -726,10 +726,18 @@
...
+    <string name="release_notes_4_2_0_title_4">Auto-refresh when a file is uploaded</string>
+    <string name="release_notes_4_2_0_subtitle_4">The file list will be now refreshed automatically when an upload whose destination folder is the one we are in is completed successfully</string>
+    <string name="release_notes_4_2_0_title_5">Download log files</string>
+    <string name="release_notes_4_2_0_subtitle_5">A new icon to download a log file to the Downloads folder of the device has been added to the log list screen</string>
```
[the remark this answers is not in the document]
Comment (Aitorbp) [reply owncloudApp/src/main/res/values/strings.xml:738]: Done

File: owncloudApp/src/main/res/values/strings.xml (hunk 5 above)
Comment (Aitorbp) [reply owncloudApp/src/main/res/values/strings.xml:734]: Done

**Review by jesmrec — APPROVED** [review]

**Pull request merged**